__tests__/feature/editing/editingLiteral.test.js

```diff
@@ -177,6 +177,25 @@ describe("editing a literal property", () => {
     expect(langBtn).toHaveTextContent("Tai languages")
   }, 25000)
 
+  it("does not show language selection when suppressed", async () => {
+    renderApp(null, history)
+
+    await screen.findByText("Uber template1", { selector: "h3" })
+
+    // Add a value
+    const input = screen.getByPlaceholderText("Uber template1, property9")
+    fireEvent.change(input, { target: { value: "foo" } })
+    fireEvent.keyDown(input, { key: "Enter", code: 13, charCode: 13 })
+
+    // There is foo text.
```

Comment:
This test isn't necessary.

Reply:
are you suggesting the entire test is not necessary, or the setup of filling in the field?  My interpretation of the reason we are doing the first part of the test of filling text into the field is to provide a unique testId for the language button to ensure the expectation (for it to be there or not) is for the field we are testing.  In this case, it is `Uber template1, property9`, which I adjusted in the fixtures to have language suppressed in order to test here.  If you change the field that is filled in, this test will fail because the language button will be shown for that field (as it should).  I adjusted the text being filled in to perhaps make this more clear.

Reply:
Sorry, L190-193 aren't necessary.

Reply:
Got it, removed.